common.sh

```diff
@@ -63,16 +63,6 @@ export SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o C
 # Verify requisites/permissions
 # Connect to system libvirt
 export LIBVIRT_DEFAULT_URI=qemu:///system
-if [ "$USER" != "root" -a "${XDG_RUNTIME_DIR:-}" == "/run/user/0" ] ; then
```

Comment:
Sorry we actually need this first check too - it will always pass when run as root, but it catches the case where someone has done `su` to a non-root user, e.g not `su -`, and in this case the XDG_RUNTIME_DIR still points at uid 0, consequently the scripts don't work as expected.

Reply:
Ah, makes sense. Reverted that part